osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/pom.xml

```diff
@@ -46,6 +79,78 @@
           </nonFilteredFileExtensions>
         </configuration>
       </plugin>
+        <!-- build helper maven plugin used for adding integration-test directories -->
+        <plugin>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 7. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=xml%3AIndentCheck)